app/src/main/java/com/blogspot/e_kanivets/moneytracker/ui/presenter/ShortSummaryPresenter.java

```diff
@@ -45,40 +46,42 @@ public ShortSummaryPresenter(Context context) {
         green = context.getResources().getColor(R.color.green);
     }
 
-    public View create(boolean shortSummary) {
+    public View create(boolean shortSummary, RecordAdapter.HeaderViewHolder mainViewHolder) {
         view = layoutInflater.inflate(R.layout.view_summary_records, null);
         view.findViewById(R.id.iv_more).setVisibility(shortSummary ? View.VISIBLE : View.INVISIBLE);
 
-        ViewHolder viewHolder = new ViewHolder(view);
-        view.setTag(viewHolder);
+        if (mainViewHolder == null)
```

Comment:
You can do `view.setTag(mainViewHolder == null ? new ViewHolder(view) : mainViewHolder);`